templates/poll/view.tpl

```diff
@@ -10,18 +10,19 @@
         </div>
         <div class="panel-body">
             <div class="poll-view-messages hidden"></div>
```

Comment:
How many messages can go in here? I'd suggest maybe showing a warning during voting step to notify users that their vote is final.

Perhaps you can put it in `poll/view/voting`

Reply:
Done see screenshot

![screen shot 2019-03-06 at 10 19 16 pm](https://user-images.githubusercontent.com/1398375/53911106-30b85100-405e-11e9-8cb0-6d50f5e37d4a.png)